lib/new-admin/config/accounts.js

```diff
@@ -13,11 +15,11 @@ const EMAIL = 'email'
 const ZERO_CONF = 'zeroConf'
 
 const ALL_ACCOUNTS = [
-  { code: 'bitpay', display: 'Bitpay', class: TICKER, cryptos: [BTC, BCH] },
-  { code: 'kraken', display: 'Kraken', class: TICKER, cryptos: [BTC, ETH, LTC, DASH, ZEC, BCH] },
-  { code: 'bitstamp', display: 'Bitstamp', class: TICKER, cryptos: [BTC, ETH, LTC, BCH] },
-  { code: 'coinbase', display: 'Coinbase', class: TICKER, cryptos: [BTC, ETH, LTC, DASH, ZEC, BCH] },
-  { code: 'itbit', display: 'itBit', class: TICKER, cryptos: [BTC, ETH] },
+  { code: 'bitpay', display: 'Bitpay', class: TICKER, cryptos: ALL['bitpay'].CRYPTO },
+  { code: 'kraken', display: 'Kraken', class: TICKER, cryptos: ALL['kraken'].CRYPTO },
+  { code: 'bitstamp', display: 'Bitstamp', class: TICKER, cryptos: ALL['bitstamp'].CRYPTO },
+  { code: 'coinbase', display: 'Coinbase', class: TICKER, cryptos: ALL['coinbase'].CRYPTO },
+  { code: 'itbit', display: 'itBit', class: TICKER, cryptos: ALL['itbit'].CRYPTO },
```

Comment:
No reason to have this `ALL['bitpay']` notation, you already know the names. `ALL.bitpay.CRYPTO` would work just fine. Maybe you want to destruct `ALL` just like you do for `COINS`